argocd_applications/managing-apps-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,69 @@
+:_mod-docs-content-type: ASSEMBLY   
+include::_attributes/common-attributes.adoc[]
+[id="managing-apps-in-non-control-plane-namespaces"]
+= Managing the application resources in non-control plane namespaces
+:context: managing-apps-in-non-control-plane-namespaces
+
+toc::[]
+
+As a cluster administrator, you can create and manage the `Application` resources in non-control plane namespaces declaratively, other than the `openshift-gitops` control plane namespace, by explicitly enabling and configuring the target namespaces in the following objects as per your requirements:
+
+* The `ArgoCD` custom resource (CR) of your user-defined cluster-scoped Argo CD instance
+* The `AppProject` custom resource (CR)
+* The `Application` CR
```

Comment:
We are mentioning the steps for configuration right below the Note. Maybe worth moving this requirements there or moving Note below the configuration.